package.json

```diff
@@ -22,6 +22,7 @@
     "docs:build": "rimraf dist && yarn docz build",
     "docs:deploy": "npm run docs:build && gh-pages -d dist",
     "prepublishOnly": "npm-run-all build",
+    "preview": "yalc publish --push && nodemon -w src -x 'yalc publish --push --changed'",
```

Comment:
You can drop the first part `yalc publish --push` as just running the `nodemon` command will run `yalc publish --push` initially.  Otherwise the command as it stands now runs twice initially.